frontend/src/Editor/QueryPanel/QueryDataPane.jsx

```diff
@@ -176,6 +182,7 @@ export const QueryDataPane = ({ darkMode, fetchDataQueries, editorRef, appId, to
                   darkMode={darkMode}
                   editorRef={editorRef}
                   appId={appId}
+                  localDs={isDataSourceLocal(query) ? 'true' : 'false'}
```

Comment:
```suggestion
                  localDs={!!isDataSourceLocal(query)}
```
We can convert output of `isDataSourceLocal` to boolean